_kits/it/progettazione-interfaccia.md

```diff
@@ -47,6 +47,10 @@ superAccordion:
       <p>Il kit è un progetto aperto e in continua evoluzione che permette, — grazie all’interazione con il team di sviluppo e la facile resa in codice con i kit di sviluppo web, in particolare con Boostrap Italia — di progettare e prototipare efficacemente l'interfaccia del touchpoint digitale in realizzazione.</p>"
       img: "/assets/icons/kit/kit_UI_InterfacciaUtente.svg"
       resources:
+        - title: Design system del Paese, nuovo UI kit v3.0.0-beta.1
```

Comment:
```suggestion
        - title: Design system del Paese, UI kit v3.0.0-beta.1
```

Reply:
È in realtà nuovo, finché sotto non togliamo l'accesso al 'vecchio' e/o non salviamo su quella repo GitHub il nuovo file .fig e i suoi esportati .sketch. Abbiamo proprio bisogno di caratterizzare nuovo e vecchio in questa fase di convivenza forzata imo.

Reply:
ok per me se è solo una questione di poco tempo